documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -36,6 +36,11 @@ include::modules/snippet_ova_tech_preview.adoc[]
 Migration using {osp} source providers only supports VMs that use only Cinder volumes.
 ====
 
+[NOTE]
+====
+Deployment of MTV on OpenShift Kubernetes Engine (OKE) is enabled. For more information, see link:https://docs.openshift.com/container-platform/4.14/welcome/oke_about.html[About OpenShift Kubernetes Engine].
+====
```

Comment:
the content of the note looks good but wouldn't it make more sense to place it in the `Software compatibility guidelines` section?

Reply:
@CarmiWisemon maybe not 'as is' - we can also say that it was added in version 2.5.3 (it's not possible to deploy earlier versions of MTV 2.5 on OKE), but yes - to that section please

Reply:
the other note looks good - no need for this one now

Reply:
@ahadas 
I added the updated note in the "Software compatibility guidelines" section.
Can we also keep the note in the "About..." section?
Thanks,
Carmi

Reply:
> Can we also keep the note in the "About..." section?

I don't think we need to keep the note in the "About..." section, deployment of OKE is not that significant - I'd say that writing about it in the software compatibility guidelines is enough

Reply:
Will do!